include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
Ah, good question. It trips over the fact that the TI processor has no byte, in fact nothing of 8 bits wide. So the calculations are off. Maybe robust isn't exactly the best way to describe it....

I must say I am not fond myself of doing this for just one quirky processor that nobody seems to be using anymore. Theoretically, other system without uint8_t or equivalent are possible; but this is the only one I've ever seen.